test/util/malicious/app_test.go

```diff
@@ -104,7 +105,7 @@ func TestMaliciousTestNode(t *testing.T) {
 
 	dah, err := da.NewDataAvailabilityHeader(eds)
 	require.NoError(t, err)
-	require.Equal(t, block.Block.DataHash.Bytes(), dah.Hash())
```

Comment:
I'm not sure if this is a bug in the cometbft fork or not but as I understood it `DataHash` and `DataRootHash` should be synonymous. 

DataHash is on the block header, and DataRootHash is on the block Data

Reply:
will create an issue to track this

Reply:
[CEL-47](https://linear.app/binarybuilders/issue/CEL-47/verify-relationship-between-datahash-and-dataroothash)